### PR TITLE
Update UWP Styles and Text Layout

### LINKED ISSUE #265: 😎 - UWP Message style adjustment
> 🟥🟩 **WINDOWS 10** - Version 1903+ is required for the UWP UI.
> 🟦🟨 [How to setup UWP](https://docs.microsoft.com/en-us/windows/uwp/get-started/get-set-up)

> 🌻 This issue is perfect for developers who are new to Miunie!

## Summary

When browsing messages in the Servers section of our UWP application, the style is a little bit off when in a light theme. The text also overflows.

The goal is to use the `Foreground="{StaticResource SystemBaseMediumColor}"` property for the `TextBlock` of user message author's name.

The `TextBlock` also needs to be adjusted to wrap user's message content instead of continuing away from the view.

## Interested? 😊 

If you'd like to help us by solving this issue, just say so in the comments. ❤️ 
And don't shy away from asking any kind of question you might have.

Should you have any questions at all, feel free to contact either me (@petrspelos) or @DraxCodes on our Discord server:

[Discord](https://discord.gg/cGhEZuk)

_And of course, thank you so much for contributing! 🎉 It's because of amazing people like yourself that Miunie exists._

### PR DESCRIPTION
**Pull request by 1n5an1ty**
Fix text-wrap bug

fixes #265 

## Changes
Add SystemBase styles to textblocks
Fix text-wrap bug

## Expected Feedback
Basic style and code review. This was a quick edit just to fix some basic styles but if you have suggestions for more styles or colors I am all ears.

## Co-Authors
@petrspelos some credit must be given as the issue outlined what was needed.

### CODE REVIEW

File: src/Miunie.WindowsApp/Views/ImpersonationChatPage.xaml [hunk 1]
```diff
@@ -17,18 +17,20 @@
     
     <Grid>
         <Grid.ColumnDefinitions>
-            <ColumnDefinition x:Name="MasterColumn" Width="320" />
+            <ColumnDefinition x:Name="MasterColumn" Width="Auto" />
             <ColumnDefinition x:Name="DetailColumn" Width="*" />
         </Grid.ColumnDefinitions>
 
-        <ListView ItemsSource="{Binding Channels}" SelectedValue="{Binding SelectedChannel, Mode=TwoWay}">
+        <ListView ItemsSource="{Binding Channels}" SelectedValue="{Binding SelectedChannel, Mode=TwoWay}" >
```
Comment (petrspelos) [inline src/Miunie.WindowsApp/Views/ImpersonationChatPage.xaml]:
```suggestion
        <ListView ItemsSource="{Binding Channels}" SelectedValue="{Binding SelectedChannel, Mode=TwoWay}">
```

File: src/Miunie.WindowsApp/Views/ImpersonationChatPage.xaml [hunk 2]
```diff
@@ -17,18 +17,20 @@
...
             <ListView.ItemTemplate>
                 <DataTemplate x:DataType="views:TextChannelView">
-                    <TextBlock Text="{Binding Name}"/>
+                    <TextBlock Text="{Binding Name}" Margin="0 0 50 0" MinWidth="100"/>
                 </DataTemplate>
             </ListView.ItemTemplate>
         </ListView>
 
+
+
```
Comment (petrspelos) [inline src/Miunie.WindowsApp/Views/ImpersonationChatPage.xaml]:
```suggestion
```

File: src/Miunie.WindowsApp/Views/ImpersonationChatPage.xaml [hunk 3]
```diff
@@ -17,18 +17,20 @@
...
+        <ListView ItemsSource="{Binding Channels}" SelectedValue="{Binding SelectedChannel, Mode=TwoWay}" >
             <ListView.ItemTemplate>
                 <DataTemplate x:DataType="views:TextChannelView">
-                    <TextBlock Text="{Binding Name}"/>
+                    <TextBlock Text="{Binding Name}" Margin="0 0 50 0" MinWidth="100"/>
                 </DataTemplate>
             </ListView.ItemTemplate>
         </ListView>
 
+
```
Comment (petrspelos) [inline src/Miunie.WindowsApp/Views/ImpersonationChatPage.xaml]:
```suggestion
```

**Comment by 1n5an1ty** [comment]
Thanks for fixing my style bugs

**Pull request merged**